docs/source/tutorials/quickstart.rst

```diff
@@ -2,145 +2,150 @@
 Quickstart
 ==========
 
-This document will lead you step by step through the necessary procedure to get
-started with manim for the first time as soon as possible.  This tutorial
-assumes you have already installed manim following the steps in
-:doc:`../installation`.
+.. note::
+ Before proceeding, install manim and make sure it's running properly by
+ following the steps in :doc:`../installation`. For
+ information on using manim with Jupyterlab or Jupyter notebook, go to the
+ documentation for the
+ :meth:`IPython magic command <manim.utils.ipython_magic.ManimMagic.manim>`,
+ ``%%manim``.
 
+Overview
+*************************************************
 
-Start a new project
-*******************
+This quickstart guide will lead you through creating a sample project using manim: an animation
+engine for precise programmatic animations.
 
-To start a new manim video project, all you need to do is choose a single
-folder where all of the files related to the video will reside.  For this
-example, this folder will be called ``project``,
+First, you will use a command line
+interface to create a ``Scene``, the class through which manim generates videos.
+In the ``Scene`` you will animate a circle. Then you will add another ``Scene`` showing
+a square transforming into a circle. This will be your introduction to manim's animation ability.
+Afterwards, you will position multiple manim objects (mobjects). Finally, you
+will learn the syntax of ``.animate``, a powerful feature that displays how
+mobjects change when you modify them with methods.
 
-.. code-block:: bash
 
-   project/
+Starting a new project
+*************************************************
 
-Every file containing code that produces a video with manim will be stored
-here, as well as any output files that manim produces and configuration files
-that manim needs.
+Start by creating a new folder. For the purposes of this guide, name the folder ``project``:
 
-.. note::
+.. code-block:: bash
 
-   In case you like to work with Jupyterlab / Jupyter notebooks, there is good news:
-   Manim ships with a ``%%manim`` IPython magic command which makes it easy to use
-   in such a setting as well. Find out more in the
-   :meth:`corresponding documentation <manim.utils.ipython_magic.ManimMagic.manim>`.
+   project/
+
+This folder is the root folder for your project. It contains all the files that manim needs to function,
+as well as any output that your project produces.
 
 
-Your first Scene
-****************
+Animating a circle
+*************************************************
 
-To produce your first scene, create a new file in your project folder called
-``scene.py``,
+1. Create a new file in your project folder named ``scene.py``.
```

Comment:
From experience with helping people on the Discord server, it seems like many new users are completely new to programming and need help with this step. Perhaps you can expand this with a little more detail, like mentioning that it's created with a code editor?

Reply:
When I was following along the quickstart guide myself, I was actually able to create and edit the requisite files using a simple text editor (Notepad). Would it still be worth noting?

Reply:
Yeah, I think so.